sklearn/utils/estimator_checks.py

```diff
@@ -123,6 +147,9 @@ def _yield_classifier_checks(name, classifier):
     yield check_classifiers_one_label
     yield check_classifiers_classes
     yield check_estimators_partial_fit_n_features
+    # test classifiers for multi-target output
+    if name not in NOT_MULTI_OUTPUT:
```

Comment:
Now that we have merged #8022, you should use tags instead.
(`if tags["multi_output"]: ...`)

You should also update `doc/developers/contributing.rst` which says `multioutput - unused for now`.

Reply:
Done. Tags are great!

Reply:
I had to add some exceptions to make this work, see [commit](https://github.com/scikit-learn/scikit-learn/pull/13392/commits/b720e066858c735223ed8a21e3631ac6233b3b55).

It appears that in case of [Lars](https://github.com/scikit-learn/scikit-learn/blob/fec7670ad07d4e639995ad005796021928344805/sklearn/linear_model/least_angle.py#L760) the parent class and some subclasses (Lars, LassoLars) support multioutput while some of it's subclasses (LarsCV, LassoLarsCV, LassoLarsIC) do not.

I removed `MultiOutputMixin` from Lars class and excluded Lars and LassoLars from [`check_supervised_y_2d`](https://github.com/scikit-learn/scikit-learn/blob/b73a51bcda362d94d8907915a382a8eb403554c8/sklearn/utils/estimator_checks.py#L1688) check as Lars and LassoLars were not throwing `DataConversionWarning` which causes `check_supervised_y_2d` to fail.

Reply:
@TomDLT do you think this would be ok?